internal/presenters/components.go

```diff
@@ -8,9 +8,58 @@ import (
 	"strings"
 	"text/template"
 
+	"github.com/charmbracelet/lipgloss"
+	"github.com/snyk/error-catalog-golang-public/snyk_errors"
 	"github.com/snyk/go-application-framework/pkg/local_workflows/json_schemas"
 )
 
+func errorLevelToStyle(errLevel string) lipgloss.Style {
+	style := lipgloss.NewStyle().
+		PaddingLeft(1).
+		PaddingRight(1).
+		Background(lipgloss.Color("1")).
+		Foreground(lipgloss.Color("15"))
+
+	if errLevel == "warn" {
+		style.
+			Background(lipgloss.Color("3")).
+			Foreground(lipgloss.Color("0"))
+	}
+
+	return style
+}
+
+func RenderError(err snyk_errors.Error) string {
+	level := strings.ToUpper(err.Level)
+
+	backgroundHighlight := errorLevelToStyle(err.Level)
+
+	var body []string
+
+	if len(err.Detail) > 0 {
+		infoLabel := lipgloss.NewStyle().Width(8).Render("Info:")
+		infoValue := lipgloss.NewStyle().PaddingLeft(1).PaddingRight(1).Width(80).Render(err.Detail)
+		body = append(body, lipgloss.JoinHorizontal(lipgloss.Top, infoLabel, infoValue))
+	}
+
+	if err.StatusCode > 0 {
+		httpLabel := lipgloss.NewStyle().Width(8).Render("HTTP:")
+		httpValue := lipgloss.NewStyle().PaddingLeft(1).PaddingRight(1).Width(80).Render(strconv.Itoa(err.StatusCode))
+		body = append(body, lipgloss.JoinHorizontal(lipgloss.Top, httpLabel, httpValue))
+	}
+
+	if len(err.Links) > 0 {
+		helpLabel := lipgloss.NewStyle().Width(8).Render("Help:")
+		helpValue := lipgloss.NewStyle().PaddingLeft(1).PaddingRight(1).Width(80).Render(strings.Join(err.Links, "\n"))
```

Comment:
Issue: If the width is set, links might get wrapped and are broken then. See below. One solution could be to remove `Width(80)` completely or increase the value.

Reply:
Flagged with design for review, ⌛ will revisit.

Reply:
Removed wrapping for link block https://github.com/snyk/go-application-framework/pull/194/commits/e397089af9096bb5991d22c6b54fa88a620df858